nico24687/le-pivot
# Tests and routes

#### What does  this PR do?
This branch includes the removal of the role column on the User table. In addition, Roles are added to newly created users in the User Controller. Most tests for Un-registered users and registered users are passing. 
#### Where should the reviewer start?
Users controller
#### How should this be manually tested?
Create a new user, then check in rails console that User.last.roles contains a registered_user Role.
#### What are the relevant story numbers?
155052319 - Authorization for multitenancy

#### Questions:
  - Do Migrations Need to be ran?
YES
  - Any other deploy steps?
you may benefit from dropping and reseeding the db, but it will be ok without it.
